spec/metadata_download_helper_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe Fastlane::Helper::MetadataDownloader do
+  describe 'downloading from GlotPress' do
```

Comment:
This test would benefit with ensuring the files are saved.

Reply:
I'm not sure I'm following, isn't it was those tests already do, by using an `in_tmp_dir` then testing `expect(File.read(destination_path_with_locale)).to eq(…)`?